app/views/shared/_sharing_widget.html.erb

```diff
@@ -1,2 +1,51 @@
+
+      <a href="http://tumblr.com/widgets/share/tool?canonicalUrl=http://teawithstrangers.com/&title=Tea%20With%20Strangers" class="sharing-tumblr sharing-button"><%= image_tag 'share/tumblr.png' %></a>
+    </div>
+  </div>
+
+</div>
+
+<script type="text/javascript">
+SHARE_COLLECTION    = []
+SHARE_COLLECTION[0] = {img: '<%= image_url 'landing/landing-graphic-redesign.jpg' %>', text: 'Tea With Strangers is all about making meaningful interactions more fun.', url: 'http://teawithstrangers.com/'}
+SHARE_COLLECTION[1] = {img: '<%= image_url 'share/share-1.png' %>', text: 'Everyone is interesting', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701568216424/?type=3' }
+SHARE_COLLECTION[2] = {img: '<%= image_url 'share/share-2.png' %>', text: 'Real people have #nofilter', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701601549754/?type=3' }
+SHARE_COLLECTION[3] = {img: '<%= image_url 'share/share-3.png' %>', text: 'You\'re always invited', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701588216422/?type=3' }
+SHARE_COLLECTION[4] = {img: '<%= image_url 'share/share-4.png' %>', text: 'New friends are just old strangers', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701541549760/?type=3'}
+SHARE_COLLECTION[5] = {img: '<%= image_url 'share/share-5.png' %>', text: 'New friends are just old strangers', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701544883093/?type=3'}
+SHARE_COLLECTION[6] = {img: '<%= image_url 'share/share-6.png' %>', text: 'Real people have #nofilter', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701584883089/?type=3' }
+SHARE_COLLECTION[7] = {img: '<%= image_url 'share/share-7.png' %>', text: 'Good vibes only', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701578216423/?type=3' }
+SHARE_COLLECTION[8] = {img: '<%= image_url 'share/share-8.png' %>', text: 'Look around', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701561549758/?type=3' }
+SHARE_COLLECTION[9] = {img: '<%= image_url 'share/share-9.png' %>', text: 'What\'s your story?', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701548216426/?type=3' }
+SHARE_COLLECTION[10] = {img: '<%= image_url 'share/share-10.png' %>', text: 'Hands were made for high-fiving', url: 'https://www.facebook.com/teawithstrangers/photos/a.818701241549790.1073741844.542640869155830/818701534883094/?type=3' }
```

Comment:
Customize: The formatting of this is really straightforward.

````javascript
{img: 'image.url', text: 'blurb text', url: 'Facebook photo URL'}
````

Don't screw it up, Ankit.